hyperbolic-rs/rudders/datasets/synopsis.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2017 The Rudders Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""File with movie lens dataset specific functions"""
```

Comment:
Add a summary of what this dataset is, and preprocessing needed to prepare it?

Reply:
Added some details but might need to create an external notebook or add a python script.